nix/modules/lanzaboote.nix

```diff
@@ -60,6 +66,7 @@ in
   
         ${cfg.package}/bin/lzbt install \
           --systemd ${pkgs.systemd} \
+          --systemd-boot-loader-config ${systemdBootLoaderConfig} \
```

Comment:
Shouldn't this end up as an extension of bootspec?

Reply:
I don't think so. A bootspec doc is generation-specific. `loader.conf`, in contrast has nothing to do with any specific generation.